File: app/src/main/java/net/tlalka/android/fiszki/models/dao/ClusterDao.java
package net.tlalka.android.fiszki.models.dao;

import com.j256.ormlite.support.ConnectionSource;
import net.tlalka.android.fiszki.models.entities.Cluster;

import java.sql.SQLException;

public class ClusterDao extends AbstractDao<Cluster, Long> {

    public ClusterDao(ConnectionSource connectionSource) throws SQLException {
        super(connectionSource, Cluster.class);
    }
}

File: app/src/main/kotlin/net/tlalka/fiszki/model/dao/ClusterDao.kt
package net.tlalka.fiszki.model.dao

import com.j256.ormlite.support.ConnectionSource
import net.tlalka.fiszki.model.entities.Cluster
import java.sql.SQLException

class ClusterDao @Throws(SQLException::class)
constructor(connectionSource: ConnectionSource) : AbstractDao<Cluster, Long>(connectionSource, Cluster::class.java)
